.github/workflows/create-azure-machines.ps1

```diff
@@ -46,128 +34,185 @@ param (
 Set-StrictMode -Version "Latest"
 $PSDefaultParameterValues["*:ErrorAction"] = "Stop"
 
-$osType = $Os.Split("-")[0]
+$AzureMatrixJson = Get-Content -Path .\.github\workflows\azure-matrix.json | ConvertFrom-Json
+$FullMatrixJson = Get-Content -Path .\.github\workflows\processed-matrix.json | ConvertFrom-Json
 
 $jobs = @()
-Write-Host "[$(Get-Date)] Creating $VMName1..."
-$jobs += Start-Job -ScriptBlock {
-    & ./.github/workflows/create-azure-machine.ps1 `
-        -VMName $Using:VMName1 `
-        -Password $Using:Password `
-        -Os $Using:Os `
-        -VMSize $Using:VMSize `
-        -ResourceGroupName $Using:ResourceGroupName `
-        -Location $Using:Location `
-        -WorkflowId $Using:WorkflowId
+$RequiredPlatforms = New-Object 'System.Collections.Generic.HashSet[string]'
+
+foreach ($entry in $AzureMatrixJson) {
+    $IdTag = $entry.runner_id
+    $Os = $entry.os
+    $VMName1 = "$IdTag-1"
```

Comment:
Could we prefix the names with `_` or something so they show up first in the list of runners (in Settings)?

Reply:
![image](https://github.com/microsoft/netperf/assets/43735701/fbc4a070-0bed-4146-ada3-070a94964706)

Reply:
We can prepend with "a" or something. Curious, why do we care that they show up first in settings? The future is we should have no static VMs at all, and the list would be purely ephemeral.

Reply:
Hmmm... I just wanted a way to easily separate them in the runner list, just in case they were still in there.

Reply:
With https://github.com/microsoft/netperf/pull/158/commits/99923c8e950a725cb5503cc72e246471297fa730, we prepend an "a" so temp VMs gets sorted on top.